release-please-config.json

```diff
@@ -11,7 +11,9 @@
       "versioning": "default",
       "extra-files": [
         "pom.xml",
-        "README.md"
+        "README.md",
+        "openfeature-provider/pom/xml",
+        "sdk-java/pom/xml"
```

Comment:
should be `<X>/pom.xml`?

Reply:
note to self: don't code from bus ever again